egs/fisher_callhome_spanish/asr1/run.sh

```diff
@@ -183,12 +196,12 @@ if [ "${stage}" -le 2 ]; then
     mkdir -p data/lang_1char/
 
     echo "make a non-linguistic symbol list"
-    cut -f 2- -d' ' "data/${train_set}/text" | tr " " "\n" | sort | uniq | grep "\[" | awk -F"]" '{print $1"]"}' | uniq > "${nlsyms}"
+    cut -f 2- -d' ' "data/${train_set_ori}/text" | tr " " "\n" | sort | uniq | grep "\[" | awk -F"]" '{print $1"]"}' | uniq > "${nlsyms}"
```

Comment:
We decided not to include `"` when it is not necessary. Could you check the diff from https://github.com/espnet/espnet/blob/v.0.3.0/egs/fisher_callhome_spanish/asr1/run.sh and follow the same style?